op-node/rollup/derive/attributes.go

```diff
@@ -181,7 +181,8 @@ func (ba *FetchingAttributesBuilder) PreparePayloadAttributes(ctx context.Contex
 	}
 
 	isVoltaTime := ba.rollupCfg.IsVolta(nextL2MilliTime / 1000)
-	pa.SetMillisecondTimestamp(nextL2MilliTime, isVoltaTime)
+	blockIntervalCount := ba.rollupCfg.MillisecondBlockInterval(nextL2MilliTime) / eth.BlockMillisecondsIntervalUint
```

Comment:
If the value is not divisible, a warning log can be printed.

Reply:
`eth.BlockMillisecondsIntervalUint` change to const